src/trading/tradingSdk.ts

```diff
@@ -12,12 +12,26 @@ import { getQuoteWithSigner } from './getQuote'
 import { postSellNativeCurrencyOrder } from './postSellNativeCurrencyOrder'
 import { getSigner, swapParamsToLimitOrderParams } from './utils'
 import { getPreSignTransaction } from './getPreSignTransaction'
+import { log } from './consts'
+import { OrderBookApi } from '../order-book'
+
+interface TradingSdkOptions {
+  logs: boolean
```

Comment:
IMO, `logs` it too generic of a term. What about `enableLogging`?

Also, should this be mentioned in the docs/readme?

Reply:
Thanks! Renamed.
I will update the docs after all fixes all together.